src/components/select/select.js

```diff
@@ -505,11 +510,10 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
 }
 
 function SelectMenuDirective($parse, $mdUtil, $mdTheming) {
-
   return {
     restrict: 'E',
     require: ['mdSelectMenu'],
-    scope: true,
+    scope: false,
```

Comment:
I lied...one more thought :flushed:

Can you add a quick comment about why this is being changed? If I recall from the Slack conversation, we don't expect it to affect anyone, but it would be good to have a comment in case another dev comes back later and wants to swap it back to `true` to fix something else.